webtransport/quic/client-indication.any.js

```diff
@@ -0,0 +1,32 @@
+// META: quic=true
+// META: script=/common/get-host-info.sub.js
+
+const PORT = 8983;
+const HOST = get_host_info().ORIGINAL_HOST;
+const ORIGIN = get_host_info().ORIGIN;
+const BASE = `quic-transport://${HOST:${PORT}/handlers`;
```

Comment:
Missing `}` after `HOST`.

Including "handlers" in the URL seems a little bit odd as it implies that you can access things in the parent directory (which would be bad if true).

Reply:
Added `}`.

Removed "handlers" from the path. In any case, the URL mapping is pretty simple and doesn't allow directory traversal (see quic_transport_server.py:130).